### ISSUE TITLE
Issue with "Using the presentation role": mostly empty sections

### ISSUE THREAD

**Issue by SjorsVanGelderen** [mdn/content#11204]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role

#### What information was incorrect, unhelpful, or incomplete?

The following sections are empty:
- Working Examples
- Notes 
- ARIA attributes used
- Related ARIA techniques 
- Compatibility
- On the page: TBD: Add support information for common UA and AT product combinations

#### Specific section or headline?
- Working Examples
- Notes 
- ARIA attributes used
- Related ARIA techniques 
- Compatibility

#### What did you expect to see?
Actual information about the topic relating to these sections.

#### Did you test this? If so, how?
The documentation for these sections is not visible on the page. I guess it's just not there.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques/using_the_presentation_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/aria_techniques/using_the_presentation_role/index.html
* Last commit: https://github.com/mdn/content/commit/510deadbe2232ed17f58b6851bc0cd809675a2b4
* Document last modified: 2021-07-09T09:20:42.000Z

</details>

**Comment by estelle** [comment]
https://developer.mozilla.org/en-US/docs/Web/accessibility/ARIA/roles/presentation_role fixes this. 
The using the ariia-presentation role linked above should disappear soon when https://github.com/mdn/content/pull/11546, which was merged today, goes live.
